docs/adr/0003-donations-and-discovery.md

```
@@ -0,0 +1,162 @@
+}
+```
+
+- **[要検証]** `RelevantEntities.updateEntities(_:for:)` / `AppEntityContext` / `SyncableEntity`
+  プロトコルの iOS バージョン（iOS 26 安定か iOS 27 か）。`IntentDonationManager`/`donate()` は
+  既存の安定 API である点はほぼ確実だが、`swiftc -typecheck` 二分岐で全体を確定する。
+
+## 設計上の核心: outbound のゲート問題と、その回避策
+
+これまでの3 ADR と違い、ドネーションは **Dart がランタイムで Swift の API を呼ぶ outbound**:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)